### ISSUE TITLE
`/availabilities` does not work

### ISSUE THREAD

**Issue by grandeljay** [grandeljay/discord-availability#13]
Even when the API does respond, the command produces an error:

```log
PHP Fatal error:  Uncaught TypeError: Grandeljay\Availability\UserAvailabilityTime::setTime(): Argument #1 ($userAvailabilityTime) must be of type int, string given, called in \path\to\discord-availability\src\Grandeljay\Availability\UserAvailability.php on line 126 and defined in \path\to\discord-availability\src\Grandeljay\Availability\UserAvailabilityTime.php:152
Stack trace:
#0 \path\to\discord-availability\src\Grandeljay\Availability\UserAvailability.php(126): Grandeljay\Availability\UserAvailabilityTime->setTime('monday 19:00')
#1 \path\to\discord-availability\src\Grandeljay\Availability\Commands\Availability.php(23): Grandeljay\Availability\UserAvailability->getUserAvailabilityTimes()
#2 \path\to\discord-availability\vendor\team-reflex\discord-php\src\Discord\Helpers\RegisteredCommand.php(98): Grandeljay\Availability\Commands\Availability->Grandeljay\Availability\Commands\{closure}(Object(Discord\Parts\Interactions\Interaction))
#3 \path\to\discord-availability\vendor\team-reflex\discord-php\src\Discord\WebSockets\Events\InteractionCreate.php(54): Discord\Helpers\RegisteredCommand->execute(Array, Object(Discord\Parts\Interactions\Interaction))
#4 \path\to\discord-availability\vendor\team-reflex\discord-php\src\Discord\Discord.php(779): Discord\WebSockets\Events\InteractionCreate->handle(Object(Discord\Helpers\Deferred), Object(stdClass))
#5 \path\to\discord-availability\vendor\team-reflex\discord-php\src\Discord\Discord.php(644): Discord\Discord->handleDispatch(Object(stdClass))
#6 \path\to\discord-availability\vendor\team-reflex\discord-php\src\Discord\Discord.php(616): Discord\Discord->processWsMessage(Object(stdClass))
#7 \path\to\discord-availability\vendor\evenement\evenement\src\Evenement\EventEmitterTrait.php(123): Discord\Discord->handleWsMessage(Object(Ratchet\RFC6455\Messaging\Message), Object(Ratchet\Client\WebSocket))
#8 \path\to\discord-availability\vendor\ratchet\pawl\src\WebSocket.php(72): Ratchet\Client\WebSocket->emit('message', Array)
#9 \path\to\discord-availability\vendor\ratchet\rfc6455\src\Messaging\MessageBuffer.php(248): Ratchet\Client\WebSocket->Ratchet\Client\{closure}(Object(Ratchet\RFC6455\Messaging\Message), Object(Ratchet\RFC6455\Messaging\MessageBuffer))
#10 \path\to\discord-availability\vendor\ratchet\rfc6455\src\Messaging\MessageBuffer.php(194): Ratchet\RFC6455\Messaging\MessageBuffer->processData('\x81~\x05\x86{"t":"INTER...')
#11 \path\to\discord-availability\vendor\evenement\evenement\src\Evenement\EventEmitterTrait.php(123): Ratchet\RFC6455\Messaging\MessageBuffer->onData('\x81~\x05\x86{"t":"INTER...')
#12 \path\to\discord-availability\vendor\react\stream\src\Util.php(71): Evenement\EventEmitter->emit('data', Array)
#13 \path\to\discord-availability\vendor\evenement\evenement\src\Evenement\EventEmitterTrait.php(123): React\Stream\Util::React\Stream\{closure}('\x81~\x05\x86{"t":"INTER...')
#14 \path\to\discord-availability\vendor\react\stream\src\DuplexResourceStream.php(196): Evenement\EventEmitter->emit('data', Array)
#15 \path\to\discord-availability\vendor\react\event-loop\src\StreamSelectLoop.php(246): React\Stream\DuplexResourceStream->handleData(Resource id #174)
#16 \path\to\discord-availability\vendor\react\event-loop\src\StreamSelectLoop.php(213): React\EventLoop\StreamSelectLoop->waitForStreamActivity(1152963)
#17 \path\to\discord-availability\vendor\team-reflex\discord-php\src\Discord\Discord.php(1440): React\EventLoop\StreamSelectLoop->run()
#18 \path\to\discord-availability\src\Grandeljay\Availability\Bot.php(91): Discord\Discord->run()
#19 \path\to\discord-availability\availability.php(20): Grandeljay\Availability\Bot->run()
#20 {main}
  thrown in \path\to\discord-availability\src\Grandeljay\Availability\UserAvailabilityTime.php on line 152

```

The default time string isn't converted to a unix timestamp. I'm pushing my fix as soon as I can interact with the bot again.

**Comment by grandeljay** [comment]
I currently can't even run the command any more. There's no output in the log so I might be temporarily blocked by the IP or something
